### PR TITLE
Added filenames to dashboard readme: Ollie

### LINKED ISSUE #283: Add file descriptions to the dashboard README
As a developer working on the StarWatch repository in the future, when I begin working with the repository, I want to have clear documentation not only on the streamlit dashboard visualisations, but also on the general functionality functionality of each of the files within the repository.

### PR DESCRIPTION
**Pull request by olliejth**
Added descriptions of dashboard visualisations. Still need to add file descriptions.

Closes #283

### CODE REVIEW

File: dashboard/README.md [hunk 1]
```diff
@@ -74,6 +74,22 @@ pytest --cov
 ```
 
 ## How it works
+#### `aurora_map.py`
+#### `azimuth_plot.py`
+#### `hold.py`
+#### `Homepage.py`
+#### `load_dashboard.py`
+#### `nasa_pipeline.py`
```
Comment (CalebMathers) [inline dashboard/README.md:82]: Could have more informative descriptions of each file in the future

File: dashboard/README.md [hunk 2]
```diff
@@ -74,6 +74,22 @@ pytest --cov
...
+
+### `/images`
+### `/shapefile`
+### `/pages`
+#### `Astronomy.py`
+#### `Stellarium.py`
+#### `Subscribe.py`
+#### `Weather.py`
```
Comment (CalebMathers) [inline dashboard/README.md:90]: Same again

**Review by CalebMathers — APPROVED** [review]

File: dashboard/README.md (hunk 1 above)
Comment (MurasatoNaoya) [reply dashboard/README.md:82]: Good to specifically mention each script needed for the dashboard, in the future it'd be nice to have each file name  hyperlink to each file for user / reader convenience.

Comment (MurasatoNaoya) [reply dashboard/README.md:82]: Good use of markdowns for consistent smaller formatting.

**Review by MurasatoNaoya — APPROVED** [review]
LGTM.

**Pull request merged**